src/project-type/context-state.ts

```diff
@@ -50,9 +51,7 @@ export function refreshAllVSCodeContext(): void {
   });
 }
 
-export async function initTagsContexts(
-  workspaceAPI: WorkspaceApi
-): Promise<void> {
+export async function initTagsContexts(workspaceAPI: BasWorkspaceApi): Promise<void> {
```

Comment:
why does our **internal** code needs to use the **read-only** interface?

Reply:
Done